gspread/models.py

```diff
@@ -606,8 +607,15 @@ def get_all_values(self):
 
             Empty trailing rows and columns will not be included.
         """
-
-        data = self.spreadsheet.values_get(self.title)
+        try:
```

Comment:
I don't think `try/except` block is needed. As far as I understand it should not be a problem always escape single quotes.